package-lock.json

```diff
@@ -5,26 +5,26 @@
   "requires": true,
   "dependencies": {
     "@actions/core": {
-      "version": "file:toolkit/actions-core-0.0.0.tgz",
+      "version": "file:E:/github/setup-go/toolkit/actions-core-0.0.0.tgz",

```

Comment:
Looks like these file paths got made into absolute paths, could you change them back to relative? We really just need to consume the toolkit from npm, just haven't gotten around to changing that yet.

Reply:
EDIT - updated to consume from npm so this no longer applies, that's why you have conflicts now though